src/resources/MachineLearning/SmartSnippetsConfiguration/SmartSnippetsConfigurationInterfaces.ts

```diff
@@ -59,28 +59,42 @@ export interface SmartSnippetsDocumentGroupPreviewParams {
 export interface SmartSnippetsDocumentGroupPreview {
     /**
      * The query that was used to fetch document information.
+     *
+     * @Example @source==("Salesforce Notifier")
      */
     query: string;
     /**
-     * Status indicating whether there are enough candidates for learning.
+     * The total number of documents in the selected sources.
```

Comment:
Was this wrongly changed by mistake?

```suggestion
     * The total number of documents in all sources.
```

Reply:
Hi! Both of these changes are intended, there is some copy change for this endpoint. See the Swagger below:
<img width="1082" alt="Screenshot 2023-11-13 at 4 21 43 PM" src="https://github.com/coveo/platform-client/assets/113382897/438e214c-9a28-4f97-a3b6-4b819eaf1e53">

`numberOfDocuments` and `numberOfDocumentsInSources` is the same property, the former is deprecated and to be replaced by `numberOfDocumentsInSources`. But to not introduce breaking change this deprecation will come later. The copy change is done to be consistent between these properties.

Reply:
ahh I see ok 👍 we should add a `@deprecated` tag in the jsdocs then

Reply:
Thanks, i used `@Deprecated` tag for these three properties